ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController.swift

```diff
@@ -227,6 +227,16 @@ class NotificationViewDemoController: DemoController {
             },
             .horizontalSpacing: .float {
                 return 5.0
+            },
+            .shadow: .shadowInfo {
+                return ShadowInfo(colorOne: DynamicColor(light: ColorValue(0xE3008C)),
```

Comment:
Nit: since these color values are used in two locations and are supposed to match, it might technically be cleaner to leverage one of the shared colors. But not a big deal.

Reply:
Wouldn't hurt!